src/core/annotation.js

```diff
@@ -602,6 +602,14 @@ var WidgetAnnotation = (function WidgetAnnotationClosure() {
       this.setFlags(AnnotationFlag.HIDDEN);
     }
 
+    // Support ResetForm Action Widget
+    if (data.fieldType === 'Btn' && dict.get('A') !== undefined) {
+      var actionRef = dict.map.A.num;
+      var xrefObject = dict.xref.fetchIfRef(dict.map.A);
```

Comment:
`.map` is an internal property of `Dict`. Use `dict.get('A')` instead of `dict.map.A` (similarly below).